cmake/externals/sentencepieceproject.ios.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
+index 8b7fb76..c08b322 100644
+--- a/src/CMakeLists.txt
++++ b/src/CMakeLists.txt
+@@ -285,10 +285,19 @@ endif()
+ list(APPEND SPM_INSTALLTARGETS
+   spm_encode spm_decode spm_normalize spm_train spm_export_vocab)
+ 
+-install(TARGETS ${SPM_INSTALLTARGETS}
+-  RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
+-  LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+-  ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR})
++if (CMAKE_SYSTEM_NAME STREQUAL "iOS")
++  install(TARGETS ${SPM_INSTALLTARGETS}
++    BUNDLE DESTINATION ${CMAKE_INSTALL_BINDIR}
```

Comment:
is it also needed on ort-extensions + onnxruntime combined build? if not, _ONNXRUNTIME_EMBEDDED flag may also should be tested.

Reply:
I think it would be needed for iOS builds in general. this applies to the sentencepiece project targets and, unless we  further specify otherwise, those should be present in either case